client/scss/overrides/_vendor.datetimepicker.scss

```diff
@@ -12,6 +12,10 @@
   box-sizing: border-box;
   display: none;
 
+  .w-dialog & {
+    z-index: theme('zIndex.dialog') + 5;
```

Comment:
I’m surprised this would work without a `calc`. Will need testing!

Reply:
You're right, without the calc it actually concatenates the string.